src/site/content/en/live/index.njk

```diff
@@ -97,26 +97,8 @@ date: 2020-06-31
 
     <div class="w-event-section__column-right w-event-section__column-right__schedule">
       {% if event.isDuringEvent or event.isPostEvent %}
-        <div class="w-event-carousel">
-          <a href="#" class="w-event-carousel__day">
-            <div class="w-event-carousel__thumbnail">
-              <img src="" alt="" width="178" height="110">
-            </div>
-            <div>Day 1 — All sessions</div>
-          </a>
-          <div class="w-event-carousel__day">
-            <div class="w-event-carousel__thumbnail">
-            </div>
-            <div>Day 2 — Coming soon</div>
-          </div>
-          <div class="w-event-carousel__day">
-            <div class="w-event-carousel__thumbnail">
-            </div>
-            <div>Day 3 — Coming soon</div>
-          </div>
-        </div>
       {% endif %}
-      {% EventTable event.days, collections.authors %}
+      {% EventTable event.days, collections.authors, event.isDuringEvent or event.isPostEvent %}
```

Comment:
I'd suggest making the carousel its own component so the EventTable doesn't have to mess with rendering it as well and we can throw it into a conditional block above the table.